tests/operations/test_cache.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import annotations
+
+import hashlib
+import pathlib
+
+import pytest
+
+import emsarray
+import emsarray.operations.cache
+
+# Sha1
+empty_sha1_hash = "da39a3ee5e6b4b0d3255bfef95601890afd80709"
+attr_hash_lon = "4151c4ac3e0e913dd2c5520c0b2c33e6f20eb004"
+str_hash = '67a0425c3bcb8a47ccc39615e59ab489d0c4b6a1'
+int_hash = '7b08e025e311c3dfcf5179b67c0fdc08e73de261'
+attr_hash_lat = "2cb433979fc2d9c3884eea8569dd6a44406950f3"
+cache_key_hash_cf1d_sha1 = "2b006999273225ed70d4810357b6a06e6bebe9a6"
+
+# Blake2b
+cache_key_hash_cf1d = "1a3226072f08441ee79f727b0775709209ff2965299539c898ecc401cf17e23f"
+cache_key_hash_cf2d = "c8d2671af35e96a08c6e33f616bc460ba3957bfa3ffc4529cfd84378e9de4c2f"
+cache_key_hash_shoc_standard = "18788cd57e4aff55dc5c0918513942bc6849d7b90e53a09bb77c4598fa3daf29"
+cache_key_hash_ugrid_mesh2d = "9e7c0ded2bfcc6843131e09303b8bec53e6edfbb18115e2c50e07adce29e8702"
+cache_key_hash_ugrid_mesh2d_one_indexed = "045be69d5ef70b20f26dc9a14e6cea8a11fe3cc9a3c4c4b303aed6f22206ac7a"
+
+
+@pytest.fixture
+def emsarray_version():
+    return '1.0.0'
+
+
+def test_hash_attributes(datasets: pathlib.Path):
+    dataset = emsarray.open_dataset(datasets / 'cfgrid1d.nc')
+
+    m_lon = hashlib.sha1()
+
+    assert m_lon.hexdigest() == empty_sha1_hash
```

Comment:
Is there a reason you test the empty hash after creating it every time? Hopefully we can trust Python sufficiently to give us an empty hash when we've just create a new instance.